lms/static/styles/components/_LMSGrader.scss

```diff
@@ -0,0 +1,35 @@
+.LMSGrader {
+  font-size: 18px;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  color: white;
+  h2, h3 {
+    margin-left: 20px;
+  }
+  ul {
+    background-color: $grey-7;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    display: flex;
+    
+    .LMSGrader__course-assignment {
```

Comment:
Nesting a class selector within a class selector generates a descendant selector (`.LMSGrader .LMSGrader__course-assignment`) which has a higher specificity that can complicate reasoning about how styles are combined. Usually we would either put each element declaration at the top level, or use:

```scss
.Component {
  &__element {
    // Styles
  }
}
```

Which will only generate a `Component__element` class rather than `.Component .Component__element`.